src/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+    datasetbuilder:
+        build: .
+        command: ["sleep", "100000000"]
```

Comment:
It's here just to keep the container up and running. We should upgrade this to the actual command but we still have to figure out how will we keep the database.